cache/cache.go

```diff
@@ -68,30 +68,35 @@ type RequestContext interface {
 }
 
 // TODO Document interface
-type CasAcCache interface {
+type BlobStore interface {
 	// TODO change to io.ReadCloser?
 	Put(kind EntryKind, hash string, size int64, rdr io.Reader, reqCtx RequestContext) error
 
 	Get(kind EntryKind, hash string, size int64, reqCtx RequestContext) (io.ReadCloser, int64, error)
 
 	Contains(kind EntryKind, hash string, size int64, reqCtx RequestContext) (bool, int64)
+}
 
+// TODO Document interface
+type AcStore interface {
```

Comment:
Is this interface used anywhere outside BlobAcStore?

Reply:
No. I’m only thinking that having a separate AcStore interface might be more generic and flexible. Especially if GetValidatedActionResult method would be extracted from disk.go in the future. Or if there would be decorators wrapping only AcStore. But I don’t have much experience of embedding interfaces and structs in golang. It might work with only BlobStore and BlobAcStore interface, if you think it is preferable to avoid AcStore?